base-template/about-base-template.md

```diff
@@ -1,8 +1,21 @@
 # About Base Template
 
+{Template author tip:
+
+* This about-base-template captures the structure and common elements that all downstream about-xxx-templates should follow.
```

Comment:
There's probably an argument for wrapping the titles in markup of some description to make sentences like this easier to read. If we do, it will need to be repo-wide, though.

Reply:
@Loquacity can you please suggest how this should be implemented. I'm unclear on what you are recommending here.
Yes, I'm expecting that after we agree on the base template, we will retrospectively go and update all our templates to align with the base template. (This would be a good first time issue for people wanting to join our project, and wanting to learn our process and how to use git.)

Reply:
Probaby the best thing to do is to use single backticks. Like this:

```suggestion
* This `about-base-template` captures the structure and common elements that all downstream `about-xxx-templates` should follow.
```